indico/modules/events/registration/templates/emails/base_registration_details.html

```diff
@@ -1,7 +1,8 @@
 {% extends 'events/registration/emails/base_registration.html' %}
 
 {% block registration_body %}
-    {% for section, fields in registration.summary_data.iteritems() %}
+    {% for section, fields in registration.summary_data.iteritems()
+       if to_managers or not section.is_manager_only %}
```

Comment:
at least we never included it in emails before.. it's manager-only information anyway